content/sensu-go/5.10/guides/scale-event-storage.md

```diff
@@ -0,0 +1,168 @@
+
+Aside from event history which is not migrated from etcd, there's no observable difference when using Postgres as the event store, and, as of this writing, neither interface supports displaying the PostgresConfig type.
+
+To verify the change was effective, take a look at the [sensu-backend log][4], where you can see that our connection to Postgres was successful:
+
+{{< highlight shell >}}
+{"component":"store","level":"warning","msg":"trying to enable external event store","time":"2019-10-02T23:31:38Z"}
+{"component":"store","level":"warning","msg":"switched event store to postgres","time":"2019-10-02T23:31:38Z"}
+{{< /highlight >}}
+
+We can also use `psql` to verify that events are being written to the `sensu_events` database:
+
+{{< highlight shell >}}
+postgres=# \c sensu_events
+You are now connected to database "sensu_events" as user "postgres".
+sensu_events=# \dt
+             List of relations
+ Schema |       Name        | Type  | Owner 
+--------+-------------------+-------+-------
+ public | events            | table | sensu
+ public | migration_version | table | sensu
+(2 rows)
+sensu_events=# select sensu_entity from events where sensu_check = 'keepalive';
+ sensu_entity 
+--------------
+ i-414141
+ i-424242
+ i-434343
+(3 rows)
+{{ /highlight }}
+
+The above illustrates connecting to the `sensu_events` database, listing the tables in the database, and requesting a list of all entities reporting keepalives.
+
+### Reverting to built-in datastore
+
+If for any reason you wish to revert to the default etcd event store, you can do so by deleting the PostgresConfig resource. In this example, my-postgres.yml contains the same configuration as we used to configure the enterprise event store earlier in this document:
+
+{{< highlight shell >}}
+sensuctl delete -f my-postgres.yml
+{{< /highlight >}}
+
+To verify that the change was effective, look for messages similar to these in the [sensu-backend log][4]:
+
+{{< highlight shell >}}
+{"component":"store","level":"warning","msg":"store configuration deleted","store":"/sensu.io/api/enterprise/store/v1/provider/postgres01","time":"2019-10-02T23:29:06Z"}
+{"component":"store","level":"warning","msg":"switched event store to etcd","time":"2019-10-02T23:29:06Z"}
+{{< /highlight >}}
+
+Similar to enabling Postgres, switching back to the etcd datastore does not migrate current event data from one store to another. You may observe old events in the web UI or  sensuctl output until such time as the etcd datastore catches up with the current state of your monitored infrastructure.
+
+_NOTE: If your Sensu Go license should expire, event storage will automatically revert to etcd._
+
+[1]:
```

Comment:
Should this be removed?

Reply:
I think I was hoping to add a link to relevant etcd documentation in the first paragraph but haven't found what I was looking for. I'll remove it for now. 👍

Reply:
```suggestion
```